src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -197,13 +197,7 @@ def _create_jobs(self, job_ids: List[str]) -> dict:
         if not job_ids:
             return {}
 
-        job_states = clients.get("execution_engine2").check_jobs(
-            {
-                "job_ids": job_ids,
-                "exclude_fields": JOB_INIT_EXCLUDED_JOB_STATE_FIELDS,
-                "return_list": 0,
-            }
-        )
+        job_states = Job.query_ee2_states(job_ids, init=True)
```

Comment:
@n1mus has consolidated most of the ee2 queries (apart from the workspace job fetch) to use the `query_ee2_state(s)` functions.